types/store/rhs.ts

```diff
@@ -17,5 +19,3 @@ export type RhsViewState = {
     isSidebarExpanded: boolean;
     isMenuOpen: boolean;
 };
-
-export type RhsState = 'mention' | 'search' | 'flag' | 'pin' | 'plugin';
```

Comment:
This variable was only used from `components/rhs_card` and from this file. Every occurrence was replaced with `ValueOf<typeof RHSStates>`